cmd/zetae2e/config/clients.go

```diff
@@ -33,25 +35,45 @@ func getClientsFromConfig(ctx context.Context, conf config.Config, account confi
 	*bind.TransactOpts,
 	error,
 ) {
+	if conf.RPCs.Solana == "" {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, fmt.Errorf("solana rpc is empty")
+	}
+	solanaClient := rpc.New(conf.RPCs.Solana)
+	if solanaClient == nil {
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, fmt.Errorf("failed to get solana client")
+	}
 	btcRPCClient, err := getBtcClient(conf.RPCs.Bitcoin)
 	if err != nil {
-		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, fmt.Errorf("failed to get btc client: %w", err)
+		return nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, nil, fmt.Errorf(
```

Comment:
nit: it would be interesting to create reusable errors, since this fmt.Errorf shares the logic between all the repetitions.

Reply:
It's okay for the tests, but I clean up the mess by returning a struct
https://github.com/zeta-chain/node/pull/2465/commits/7032ee7e171261373ecf98860bd981ce453e310b